.golangci.yml

```diff
@@ -57,11 +57,11 @@ linters-settings:
       - cancelled
   lll:
     # Default is 120. '\t' is counted as 1 character.
-    # set our project to 200, as things like v3_structs with inline comments end up being a touch long
+    # set our project to 210, as things like v3_structs with inline comments end up being a touch long
```

Comment:
just curious, why this linter update?

Reply:
I removed this change with my latest changes as it is no longer needed